### PR TITLE
Add metadata endpoints to docs and fix sphinx warnings

### PR DESCRIPTION
**Pull request by amCap1712**
Add metadata blueprint, explore api blueprint, examples for metadata api and feedback api to docs.

Building docs had these warnings:

1) WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid langauge code. Falling back to 'en' (English).

To fix, explicitly set language.

2) WARNING: html_static_path entry '_static' does not exist

Add _static directory to git. also need a file so that git tracks this directory so moved favicon.ico inside this folder.

3) WARNING: Bullet list ends without a blank line; unexpected unindent.

Fixed indents in code comments.

### CODE REVIEW

**Review by mayhem — APPROVED** [review]
Thanks so much for adding this!

File: docs/users/api-usage.rst [hunk 1]
```diff
@@ -65,7 +65,83 @@ of tracks like this:
...
+artist name. For instance,
+
+.. include:: ./api_usage_examples/lookup_metadata.py
+   :code: python
+
+.. code-block:: bash
+
+  ./lookup_metadata.py
+
+  Please input the track name of the listen: One Last Time
```
Comment (mayhem) [inline docs/users/api-usage.rst:83]: I find this block confusing, but that may only be because of how it is displayed in a PR. I wonder if stating it like this might be less confusing:

Please provide the prompted data to the script to lookup the given track. Currently the release argument is not used, but we plan to support in the near future, so we encourage you to start sending release information if you have it.

File: docs/users/api-usage.rst [hunk 2]
```diff
@@ -65,7 +65,83 @@ of tracks like this:
...
+for an example of the same. Here is an example of how to submit love/hate feedback using
+the ListenBrainz API. Refer to :ref:`feedback-api` for more details.
+
+.. include:: ./api_usage_examples/submit_feedback.py
+   :code: python
+
+.. code-block:: bash
+
+  ./submit_feedback.py
+  Please input the recording mbid of the listen: 70c75409-4224-4bab-836a-eb5f3a9c31d3
```
Comment (mayhem) [inline docs/users/api-usage.rst:140]: Same thing might apply here, but I'm not really sure.

**Review by mayhem — APPROVED** [review]
I suppose this is all ok -- hard to follow given how things look in a PR. Feel free to ignore my comments if they don't resonate.

File: docs/users/api-usage.rst (hunk 1 above)
Comment (amCap1712) [reply docs/users/api-usage.rst]: Done.

File: docs/users/api-usage.rst (hunk 2 above)
Comment (amCap1712) [reply docs/users/api-usage.rst]: Done.
